src/test/java/emissary/test/core/junit5/RegressionTestUtil.java

```diff
@@ -95,6 +96,8 @@ public final class RegressionTestUtil {
      */
     private static final Path TEST_RESX = getTestResx();
 
+    private static AtomicReference<Class<?>> answerFileClassRef = null;
```

Comment:
This isn't thread-safe

Reply:
I altered it so the answerFileClasRef is never stored within RegressionTestUtil. I'm not sure if we want to keep the methods and add duplicates with different signatures, or if this is the preferred option.

Reply:
I think this change is fine.  If we do any further cleanup, I'd probably start again in `UnitTest` and remove the `AtomicReference<?>` wrapper, and get the `Ref` suffix off the variable and parameter names.  There's probably a bit more ceremony than it actually needs.  This essentially private to RegressionTest -> RegressionTestUtil, so I'm not currently concerned if it isn't the most intuitive interface.